data/human/culture conversations.txt

```diff
@@ -149,6 +163,49 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Curious Waiter"
+	minor
+	source
+		government "Syndicate"
+		attributes "fatory"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`On a whim, you decide to dine in at a restaurant well clear of the busy spaceport. You swiftly find yourself searching up a list of delicacies on a physical menu as people stream past the window by your booth, each looking drained from the long work day. Meal soon ordered, you continue watching the crowd until it arrives, a local version of a katsu Don and dig in.`
+			`	The place is quiet, perhaps a little early for the dinner rush yet, and you are half way through your meal when a waiter stops beside you, coughing to catch your attention.`
+			choice
+				`	"May I help you?"`
+				`	"Yes?"`
+				`	"What do you want?"`
+			`	"Sorry," he says, grabbing the edges of his shirt. "But... may I ask... what's it like to see the stars? To visit other worlds?"`
+			choice
```

Comment:
All these choices seem to be pretty neutral or negative-leaning. I think a little variety from adding in a more positive response or two would be appreciated. It wouldn't be too much of a stretch to add in an alternative response from the waiter too for said positive choice.

Reply:
@MasterOfGrey ^

Reply:
I genuinely could not think of a thing to say, that would be culturally and factually appropriate to human space, that wouldn't also count as encouraging personal endangerment to the guy who I envisioned to be obviously young and impressionable.
I'm not beholden to how it works now, but I have no ideas for that unless we re-frame it.

Reply:
Does the player have to be a constant pillar of factuality and responsibility, though? The player character is meant to be a reflection of the real-world player, and they might want to give a more positive (or negative), but less accurate description of human space, whether it be to not disappoint the waiter or because they perceive it as better than it actually is.

Reply:
> Does the player have to be a constant pillar of factuality and responsibility, though? The player character is meant to be a reflection of the real-world player, and they might want to give a more positive (or negative), but less accurate description of human space, whether it be to not disappoint the waiter or because they perceive it as better than it actually is.

They don't. The main idea for this one was more as a passing note. It's like asking a soldier what it's like to fight on the frontlines.

Reply:
Was `May I help you?` always there? Seems like that fits the bill of what I was asking for here. Huh. Not sure how I didn't see that

Reply:
I think it's been there the whole time.

Reply:
Lol whoops